src/components/SearchForm.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Field, Form, withFormik } from 'formik';
+import * as Yup from 'yup';
+
+const SearchForm = ({
+  values,
+  errors,
+  touched,
+}) => (
+  <Form>
+    <div className="form-field">
+      <label>Include these words or phrases
+        <Field
+          type="text"
+          name="searchTerms"
+          placeholder="seperate search terms with a comma (,)"
+        />
+      </label>
+      {touched.searchTerms && errors.searchTerms &&
+      <p className="form-error">{errors.searchTerms}</p>}
+    </div>
+    <div className="form-field">
+      <label>Location (optional, zipcode)
+        <Field
+          type="text"
+          name="location"
+          placeholder="97015"
+        />
+      </label>
+      {touched.location && errors.location &&
+      <p className="form-error">{errors.location}</p>}
+    </div>
+    <div className="form-field">
+      <label>radius (in miles)
+        <Field
+          type="text"
+          name="radius"
+          placeholder="25"
+        />
+      </label>
+      {touched.location && errors.radius &&
+      <p className="form-error">{errors.radius}</p>}
+    </div>
+    <div className="form-field">
+      <label>Timeframe (optional)
+        <Field
+          type="text"
+          name="timeFrame"
+          placeholder="1 month"
```

Comment:
Is this going to get translated into a duration that we can use in a query (e.g. get all posts more recent than one month ago)?

How will this time frame input get translated into a date?